src/main/webapp/index.html

```diff
@@ -1,26 +1,40 @@
 <!DOCTYPE html>
 <html>
     <head>
-    <!-- Required meta tags -->
-        <meta charset="utf-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1">
-
-        <!-- Bootstrap CSS -->
-        <link 
-            href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css" 
-            rel="stylesheet" 
-            integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" 
-            crossorigin="anonymous">
-        
-        <script 
-            src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/js/bootstrap.bundle.min.js" 
-            integrity="sha384-JEW9xMcG8R+pH31jmWH6WWP0WintQrMb4s7ZOdauHnUtxwoG2vI5DkLtS3qm9Ekf" 
-            crossorigin="anonymous">
-        </script>
-
+        <link rel="stylesheet" href="style.css">
         <title>Munera</title>
     </head>
     <body>
-        <a href="home_page.html"><button type="button" class="btn btn-primary">Get Started</button></a>
+        <ul class="cb-slideshow">
+            <li>
+                <span></span>
+            </li>
+            <li>
+                <span></span>
+            </li>
+            <li>
+                <span></span>
+            </li>
+            <li>
+                <span></span>
+            </li>
+            <li>
+                <span></span>
+            </li>
+            <li>
+                <span></span>
+            </li>
```

Comment:
This section could benefit from some comments to show that it isn't just a list of empty elements